apps/web/app/routes/examples/forms/multiple-forms.tsx

```diff
@@ -0,0 +1,146 @@
+import hljs from 'highlight.js/lib/common'
+import type {
+  ActionFunction,
+  LoaderFunction,
+  MetaFunction,
+} from '@remix-run/node'
+import { formAction } from '~/formAction'
+import { z } from 'zod'
+import Form from '~/ui/form'
+import { metaTags } from '~/helpers'
+import { makeDomainFunction } from 'domain-functions'
+import Example from '~/ui/example'
+
+const title = 'Multiple Forms'
```

Comment:
We use `Sentence case` instead of `Title Case` for our titles. Can you change it to `Multiple forms`? Here and on the menu.